packages/core/src/browser/terminal/terminal-model.ts

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (C) 2018 Red Hat, Inc. and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import { Disposable } from "../../common";
+
+/**
+ * Terminal model describes interfaces for creation and using terminal widget.
+ */
+
+/**
+ * Terminal widget options.
+ */
+export const TerminalWidgetOptions = Symbol("TerminalWidgetOptions");
+export interface TerminalWidgetOptions {
+
+    /**
+     * Human readable terminal representation on the UI.
```

Comment:
For each option, it might be good to say what they default to, if they are not specified.

Reply:
Ok, I will apply it.